### PR TITLE
noship feature for function that did not make 1.0

### PR DESCRIPTION
**Pull request by njr-11**
Create a nonship vNext feature where we can move function that did not make it into Data 1.0 that is likely to be added in the next version. Specifically, this is where we will end up moving the annotations for annotative query definition, but we will need to so some refactoring in test buckets first which will require some additional pull requests.

### CODE REVIEW

**Comment by njr-11** [comment]
#build

File: dev/com.ibm.websphere.appserver.features/visibility/public/data-1.1/resources/l10n/io.openliberty.data-1.1.properties [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+#
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+#
+
+description=Jakarta Data provides a repository-based data access pattern where the application defines custom data repository interfaces and the container/runtime provides the implementation.
```
Comment (dmuelle) [inline dev/com.ibm.websphere.appserver.features/visibility/public/data-1.1/resources/l10n/io.openliberty.data-1.1.properties]:
```suggestion
description=Jakarta Data provides a repository-based data access pattern where the application defines custom data repository interfaces and the container or runtime provides the implementation.
```

**Review by dmuelle — DISMISSED** [review]
LGTM

**Comment by njr-11** [comment]
#build
#libby

**Review by KyleAure — DISMISSED** [review]

**Comment by njr-11** [comment]
#build
#libby

**Review by KyleAure — DISMISSED** [review]

**Comment by njr-11** [comment]
#build
#libby

**Review by KyleAure — DISMISSED** [review]

**Comment by njr-11** [comment]
#build
#libby

**Review by KyleAure — APPROVED** [review]

**Pull request merged**